Concepts section overhaul (mirror project) (#296)
pinot-contrib/pinot-docs: basics/components/README.md

  processing and analytics. Unleash the power of Pinot's building blocks for
  high-performance data-driven applications.
---
Apache Pinot™ is a database designed to deliver highly concurrent, ultra-low-latency queries on large datasets through a set of common data model abstractions. Delivering on these goals requires several foundational architectural commitments, including:

* Storing data in columnar form to support high-performance scanning
* Sharding of data to scale both storage and computation
* A distributed architecture designed to scale capacity linearly
* A tabular data model read by SQL queries

# Components

> Pages in this section define and describe the major components and logical abstractions used in Pinot. 

Reviewer: ```suggestion
Learn about the major components and logical abstractions used in Pinot. 
```
